core/css/decanter.css

```diff
@@ -300,139 +300,187 @@ template {
 
```

Comment:
Scott and John would prefer to load all fonts that are available on Google from Google. We should only be hosting fonts that aren't available from Google.

An open question is where / how to host them. If we just bundle them into Decanter, then people building sites on Decanter will have to deploy them to every site. I think it would be better to host them on Stanford's homesite, and have all Decanter-based sites refer to a single source. I've put Stanford's ligature font (for the wordmark) in https://www.stanford.edu/assets/fonts/Stanford.ttf .

If you could tell me what fonts aren't available from Google, I'll put them in the same directory. Then you can update Decanter to load them from there.

Reply:
All of the fonts pictured below are now available in https://www-media.stanford.edu/assets/fonts/ : 

![fonts available on homesite](https://user-images.githubusercontent.com/5298452/47876352-55fa3180-ddd6-11e8-9ed4-62fc75a314dd.png)

Reply:
Thanks, JB!